server/src/bootstrap/index.ts

```diff
@@ -128,6 +132,31 @@ const answersService = new AnswersService({ Post, Answer })
 const topicsService = new TopicsService({ Topic })
 const userService = new UserService({ User, Tag, Agency })
 
+const searchHost = searchConfig.host
+const protocol = 'https'
+const port = searchConfig.port
+const auth = `${searchConfig.username}:${encodeURIComponent(
+  searchConfig.password,
+)}`
+let connectionOptions
+if (baseConfig.nodeEnv === Environment.Dev) {
+  connectionOptions = { rejectUnauthorized: false } // Turn off certificate verification (rejectUnauthorized: false)
+} else {
+  connectionOptions = {
+    ca: fs.readFileSync('/etc/ssl/cert.pem'), // Path to CA certificate for Alpine Linux, which is the OS AskGov runs on
+  }
+}
+const client = new Client({
+  node: protocol + '://' + auth + '@' + searchHost + ':' + port,
+  ssl: connectionOptions,
+})
```

Comment:
I would put this into `bootstrap/search.ts`, similar to how we bootstrap the Sequelize client. Could you file a PR for this please?

Reply:
Ah sure 👍